.github/workflows/android.yml

```diff
@@ -12,26 +12,33 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
+
       - uses: actions/setup-node@v1
         with:
-          node-version: '16.1.0'
+          node-version: '14.16.1'
```

Comment:
could we here move again up to 16.1.0? or are there any other reason to use v14?

Reply:
Indeed, I think we can upgrade again, the downgrade was just to test if this was causing crashes. Done in 3b5bf5bd